plugins/linear-genome-view/src/BaseLinearDisplay/models/serverSideRenderedBlock.ts

```diff
@@ -189,7 +189,11 @@ function renderBlockData(self: Instance<BlockStateModel>) {
       let matchFound = false
       assemblyNames.forEach((assemblyName: string) => {
         const assembly = assemblyManager.get(assemblyName)
-        if (assembly && assembly.aliases.includes(self.region.assemblyName))
+        if (
+          assembly &&
+          (assembly.name === self.region.assemblyName ||
+            assembly.aliases.includes(self.region.assemblyName))
+        )
```

Comment:
Is assembly a proper object here?  Could we make a method on assemblies called something like `hasName`, so this check would be `assembly && assembly.hasName(self.region.assemblyName)`?

Reply:
I bet there are other places in the code where this check is being done, probably want to grep for those

Reply:
Could do assembly.allAlliases.includes(name) or make helper method